src/OpenLoco/src/Objects/SteamObject.h

```diff
@@ -46,7 +46,7 @@ namespace OpenLoco
         const ImageAndHeight* frameInfoType0; // 0x16
         const ImageAndHeight* frameInfoType1; // 0x1A
         uint8_t numSoundEffects;              // 0x1E
-        SoundObjectId_t soundEffects[9];      // size tbc
+        SoundObjectId_t soundEffects[9];      // 0x1F
```

Comment:
are we sure then on the size?

Reply:
ah apologies, I though this was meant to be the offset, not the actual size of the array. `SoundObjectId_t` is  `uint8_t`, so this would just be `9` if you want the size

Reply:
oh i see, that makes sense. i know this isn't proof, but the only vanilla object with *any* sound ids is `STEAM` which does have 8. all the vanilla steam objects have that 9th byte as 0 as well, so it's either unused, or otherwise set at load time, which since there doesn't seem to be any loading code for anything, i'd say is unlikely